src/main/java/core/basesyntax/model/User.java

```diff
@@ -1,10 +1,24 @@
 package core.basesyntax.model;
 
 import java.util.List;
-
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "user")
 public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String username;
```

Comment:
same

Reply:
Agree everywhere but here - it is like login: one word, isn't it?